static/src/js/components/Errors/ErrorBoundary.jsx

```diff
@@ -71,11 +71,11 @@ class ErrorBoundary extends Component {
             {' '}
             when contacting
             {' '}
-            <a href="mailto:support@earthdata.nasa.gov">Earthdata Operations</a>
+            <a href="mailto:support@earthdata.nasa.gov" style={{ textDecoration: 'underline dashed' }}>Earthdata Operations</a>
             .
           </p>
           <p>
-            <a href="/">Click here</a>
+            <a href="/" style={{ textDecoration: 'underline dashed' }}>Click here</a>
```

Comment:
we try to avoid inline styling  https://wiki.earthdata.nasa.gov/display/EDSC/Developer%27s+Guide#Developer'sGuide-CSSwithSass